packages/perseus/src/widgets.ts

```diff
@@ -162,7 +162,12 @@ export const getVersionVector = (): {
 
 export const getPublicWidgets = (): Record<string, WidgetExports> => {
     return widgets.entries().reduce((acc, [key, value]) => {
-        if (!value.hidden) {
+        /**
+         * Even though we don't want content creators adding new "hidden" widgets,
+         * we still have to maintain editors for hidden widgets in order to support
+         * old content. So this lets us use hidden widgets in Storybook.
+         */
+        if (process.env.STORYBOOK || !value.hidden) {
```

Comment:
Greatly appreciate this, and the commenting! It would be nice to have a way to swap the storybook view between "all" and "public" modes while testing the editors, but that's something we can worry about later if it comes up.